simulator/examples/chaining.rs

```diff
@@ -51,7 +51,7 @@ fn main() -> Result<(), core::convert::Infallible> {
         )
         .chain(
             // Draw a square with a 3px wide stroke.
```

Comment:
I missed this previously. Not really related, but could you fix it as part of this PR anyway?

```suggestion
            // Draw a square with a 3px wide stroke.
```

Reply:
Yes.